src/library/scala/deprecated.scala

```diff
@@ -11,11 +11,55 @@ package scala
 import scala.annotation.meta._
 
 /** An annotation that designates that a definition is deprecated.
- *  Access to the member then generates a deprecated warning.
+ *  A deprecation warning is issued upon usage of the annotated definition.
  *
+ *  Library authors should state the library's deprecation policy in their documentation to give
+ *  developers guidance on how long a deprecated definition will be preserved.
+ *
+ *  Library authors should prepend the name of their library to the version number to help
+ *  developers distinguish deprecations coming from different libraries:
+ *
+ *  {{{
+ *  @deprecated("this method will be removed", "FooLib 12.0")
+ *  def oldMethod(x: Int) = ...
+ *  }}}
+ *
+ *  The compiler will emit deprecation warnings grouped by library and version:
+ *
+ *  {{{
+ *  oldMethod(1)
+ *  oldMethod(2)
+ *  aDeprecatedMethodFromBarLibrary(3, 4)
+ *
+ *  // warning: there were two deprecation warnings (since FooLib 12.0)
+ *  // warning: there was one deprecation warning (since BarLib 3.2)
+ *  // warning: there were three deprecation warnings in total; re-run with -deprecation for details
+ *  }}}
+ *
+ *  The Scala language and its standard library will only add new deprecations on major releases to
+ *  avoid the introduction of new warnings after minor version upgrades of Scala.
```

Comment:
Mhh, ok. I only wanted to document the existing policy. (I tried to add minor deprecations in the past and was told, that I couldn't do that.)

Should I drop this bit?